libs/designer/src/lib/ui/templates/review/ReviewExisting.tsx

```diff
@@ -31,7 +33,12 @@ export const ReviewExisting = ({ name, resourceOverrides }: ReviewExistingProps)
           {name}
         </Text>
       </div>
-
+      {description && (
+        <div className="msla-templates-review-block">
+          <Text>{resourceOverrides?.workflowDescription ?? resourceStrings.DESCRIPTION}</Text>
+          <Text weight="semibold">{description}</Text>
+        </div>
+      )}
```

Comment:
NO description here and we actually do not have a description for workflow today